test/Testexport.jl

```diff
@@ -0,0 +1,20 @@
+module TestexportSpec
+
+using Test
+using DataFrames
+using CSV
+
+const PATH = "data/output"
+const TAG = "alcap-example-deposition_only"
+
+data = CSV.read("$(PATH)/$(TAG)_sc.csv", DataFrame)
+sc1_f1 = data.var"sc1_f1 [m]"
+sc1_f2 = data.var"sc1_f2 [m]"
+sc1_f3 = data.var"sc1_f3 [m]"
+sum(sc1_f1+sc1_f2+sc1_f3)
+
+data2 = CSV.read("$(PATH)/$(TAG)_sac.csv", DataFrame)
+sac1 = data2.var"sac1 [m]"[end]
+
+@test sum(sc1_f1+sc1_f2+sc1_f3) ≈ sac1
```

Comment:
these quantities were never meant to be equal.

From what I understand, the stratigraphic column gives you the amount and type of sediment for each time step (basically: production corrected for later erosion), whereas the sediment accumulation curve is an integral quantity (slight modification of the age depth model).

See https://github.com/MindTheGap-ERC/CarboKitten.jl/blob/b6a50362b6c2cd02b5de7e060fbb8114141e402f/src/Export.jl#L174 to see how ADM and SAC are related.